test/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,5 @@
+class UserMailerPreview < ActionMailer::Preview
```

Comment:
I'm not familiar with ActionMailer::Preview, how is this different from the letter_opener gem we already use?
Based on this article it seems we may want to stick with letter opener. https://dev.mikamai.com/2017/03/30/setting-an-environment-for-previewing-email-in-rails/

Reply:
I didn't realize that the letter_opener gem was part of the project's dependencies. I'll get rid of ActionMailer.